src/NerdMinerV2.ino.cpp

```diff
@@ -1,8 +1,13 @@
+//Botón configuración
+#define PIN_BUTTON_1 0
+#define PIN_BUTTON_2 14
```

Comment:
You should leave these lines in wManager.h

Reply:
Done